README.md

```diff
@@ -11,7 +11,9 @@
   · SCSS
   · Less
   · JSX
+  · HTML
   · Vue
+  · Angular
   · GraphQL
   · JSON
   · Markdown
```

Comment:
This line is starting to get a little long. Should it be reformatted somehow?

Reply:
IMO, JavaScript should be the first one, sorting by group should be better.

Reply:
We can have **JavaScript** in bold :)

Reply:
WDYT? https://github.com/prettier/prettier/blob/b944ad58eed30d0dc7abc347e744dbfbe919891d/README.md#opinionated-code-formatter